### DISCUSSION TITLE
How to have a border around your column/row?

### DISCUSSION THREAD

**Discussion by metaperl** [zauberzeug/nicegui#1666] [Q&A]
### Question

I would like the 2 elements in this `ui.row()` call to be visually grouped with a border that goes around both elements:

```python
            with ui.column():
                ui.label(_[0].upper())
                selects.append(
                    ui.select(_, on_change=lambda e: add_feeling(e.value))
                )
```

What arguments can I supply to `ui.row()` so that it puts a border around the `label` and `select`?

# here are before and after screenshots

## before

![image](https://github.com/zauberzeug/nicegui/assets/21293/7e20812e-56a9-46ee-b8a0-4e433711e6a3)

## after ( i only put borders around 4 elements but you get the idea)

![image](https://github.com/zauberzeug/nicegui/assets/21293/76196581-b204-490b-8a2b-4735e49e0c56)

**Comment by natankeddem** [comment]
Even though it isn't in the documentation it looks like `ui.select` is based off of Quasar QSelect. So you could use `props` to modify the styling:
```py
from nicegui import ui

select1 = ui.select([1, 2, 3], value=1).props("square outlined")

ui.run()
```
More information and styling demos here:
https://quasar.dev/vue-components/select/

**Reply by metaperl** [reply]
I think `ui.card` will suffice.
